common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -564,78 +564,116 @@ def test_navigation_links(self):
 
 
 @attr('shard_5')
-class CreateTeamTest(TeamsTabBase):
+@ddt.ddt
+class CreateAndEditTeamTest(TeamsTabBase):
     """
     Tests for creating a new Team within a Topic on the Teams page.
     """
 
     def setUp(self):
-        super(CreateTeamTest, self).setUp()
+        super(CreateAndEditTeamTest, self).setUp()
         self.topic = {'name': 'Example Topic', 'id': 'example_topic', 'description': 'Description'}
-        self.set_team_configuration({'course_id': self.course_id, 'max_team_size': 10, 'topics': [self.topic]})
+        self.set_team_configuration(
+            {'course_id': self.course_id, 'max_team_size': 10, 'topics': [self.topic]},
+            global_staff=True
```

Comment:
Instead of using a global staff user for all the tests, it would be better to use a course staff user.